apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/chronoLocalDateTimeAssertions.kt

```diff
@@ -74,3 +74,68 @@ fun <T : ChronoLocalDateTime<out ChronoLocalDate>> Expect<T>.isAfterOrEqual(
 fun <T : ChronoLocalDateTime<out ChronoLocalDate>> Expect<T>.isEqual(
     expected: ChronoLocalDateTime<*>
 ): Expect<T> = _logicAppend { isEqual(expected) }
+
+/**
+ * Expects that the subject of the assertion (a [ChronoLocalDateTime])
+ * is before the [expected] [String].
```

Comment:
Please also add to the other KDoc
```suggestion
 * is before the [expected] [LocalDateTime] given as [String] in (modified) ISO 8601 format.
 * 
 * The string will be converted to a LocalDateTime according to ISO 8601 but with a slight deviation. The alternative notation (e.g. 20200401120001 instead of 2020-04-01T12:00:01) is not supported and we accept a date without time in which case 00:00:00 is used. Following the supported formats from the longest to the shortest:
* * yyyy-mm-ddThh:mm:ss.sss (up to 9 digits for nanoseconds)
* * yyyy-mm-ddThh:mm:ss
* * yyyy-mm-ddThh:mm
* * yyyy-mm-dd
```